src/webserver/WebServer.jl

```diff
@@ -67,6 +67,7 @@ function run(; kwargs...)
     options = Configuration.from_flat_kwargs(; kwargs...)
     run(options)
 end
+precompile(run, ())
```

Comment:
This one works. Although I cannot measure `Pluto.run()` easily, I can measure `Configuration.from_flat_kwargs` which is the biggest source of latency according to SnoopCompile

## Without `precompile(run, ())`

```julia
$ julia --project -ie 'using Pluto'
...

julia> @time @eval Pluto.Configuration.from_flat_kwargs();
  2.158083 seconds (3.60 M allocations: 194.337 MiB, 7.19% gc time, 99.96% compilation time)
```

## With `precompile(run, ())`

```julia
julia> @time @eval Pluto.Configuration.from_flat_kwargs();
  1.431652 seconds (2.64 M allocations: 143.595 MiB, 11.35% gc time, 99.92% compilation time)
```